drivers/virtualbox/vbm.go

```diff
@@ -56,6 +56,19 @@ func setVBoxManageCmd() string {
 	return cmd
 }
 
+// VBoxManager defines the interface to communicate to VirtualBox.
+type VBoxManager interface {
+	queryVirtualBox(args ...string) (string, string, error)
```

Comment:
Nitpick:  I'd call this `vbmOutErr` for consistency with the package-level function and minimal search-and-replace effort.